Source/Blazorise/Enums/TextDecoration.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Blazorise;
+
+/// <summary>
+/// Defines the text decoration.
+/// </summary>
+public enum TextDecoration
+{
+    /// <summary>
+    /// No decoration. The text renders as it is. This is default.
+    /// </summary>
+    Default,
```

Comment:
I'm not sure how to better describe this.
But it can still inherit decoration from somewhere else can it not?

The thing is that default, basically does nothing, it does not apply any special css class, is this not right?

Saying that it **renders the text as it is** might be misleading I'm not sure.

Reply:
I can add the comment saying that it will use rule from CSS framework, or inherited from the parent element.